src/families/ethereum/modules/erc20.js

```diff
@@ -38,6 +38,7 @@ const erc20approve: ModeModule = {
     if (!t.useAllAmount && t.amount.eq(0)) {
       result.errors.amount = new AmountRequired();
     }
+    result.amount = t.useAllAmount ? BigNumber(2).pow(256).minus(1) : t.amount;
```

Comment:
just use t.amount here the UI handles the no limit

Reply:
`BigNumber(2).pow(256).minus(1)` isn't it a constant somewhere?

Reply:
I think we haven't set it up but I will